code/src/Grid/IGridWijmo.ts

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace Grid {
+    export interface IGridWijmo extends IGridGeneric<wijmo.grid.FlexGrid> {}
+}
```

Comment:
This interface was left outside of the @types folder on purpose? For being provider dependent?

Reply:
Also, what for are we actually using it?

Reply:
I think you are talking about the Interfaces folder... well for this reason I broke this task in Parts...
This first one, I tried to follow the folders we already have... and we do not have a InterfacesFolder, but we do have a InterfacesFile with a lot of interfaces
The AbstractConfiguration is off any folder just because it is used by Columns and Grids... I left this one on a neutral territory...